### ISSUE TITLE
Research Using React-Hot-Loader

### ISSUE THREAD

**Issue by devloco** [devloco/create-react-wptheme#25]
This looks like the latest work:
https://github.com/facebook/create-react-app/pull/5958

And here's the doc site:
http://gaearon.github.io/react-hot-loader/

**Comment by benjaminviaud** [comment]
Hello,

I come with you to know if you have implement the React Hot Loader or if you say me how to implement it ? I try to use it but it doesn't work.

![image](https://user-images.githubusercontent.com/25077689/73061053-8de90b00-3e99-11ea-9f11-5111d6c204b5.png)

Thanks for your work.

Ben

**Comment by devloco** [comment]
Hi @benjaminviaud,

It looks like `react-hot-loader` is obsolete now, replaced by [Fast Refresh](https://github.com/WebHotelier/webpack-fast-refresh). See this post:

[How should we set up apps for HMR now that Fast Refresh replaces react-hot-loader?](https://github.com/facebook/react/issues/16604)

In particular, see Dan Abramov's comment on that post:
https://github.com/facebook/react/issues/16604#issuecomment-528663101

**And, purely my opinion here**, but I've used `react-hot-loader` in other projects, and I don't like it at all. It's untrustworthy. You always need a full browser refresh to be absolutely sure what you're seeing on screen is what your visitors/customers will see when the project is deployed. And once I realized that I was always manually refreshing anyway, I've never used it again.

I'm sure it makes for a cool demo. But for getting work done, I avoid it.

Anyway, if Facebook ever officially adds support for Fast Refresh (or any other tool), then `Create React WPTheme` will also get that feature (At this point `create-react-wptheme` is using a copy of the webpack.config.js file used by `create-react-app` with only the smallest changes (e.g. support for PHP files, etc.)).
